libmimalloc-sys fails to build for wasm target on Windows

Not sure if this allocator is intended to also work for wasm, but in my frontend I need speed more than size (it's a UI for a desktop application), so I thought I'd try it..
But it fails to build for wasm:
```
error[E0432]: unresolved imports `libc::c_void`, `libc::size_t`
 --> C:\Users\me\.cargo\registry\src\github.com-1ecc6299db9ec823\libmimalloc-sys-0.1.9\src\lib.rs:3:12
  |
3 | use libc::{c_void, size_t};
  |            ^^^^^^  ^^^^^^ no `size_t` in the root
  |            |
  |            no `c_void` in the root

error: could not compile `libmimalloc-sys`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,6 @@ members = ["libmimalloc-sys" ]
 travis-ci = { repository = "purpleprotocol/mimalloc_rust" }
 
 [dependencies]
-libc = "0.2"
 libmimalloc-sys = { path = "libmimalloc-sys", version = "0.1.10" }
 
 [features]

--- a/libmimalloc-sys/Cargo.toml
+++ b/libmimalloc-sys/Cargo.toml
@@ -10,7 +10,6 @@ description = "Sys crate wrapping the mimalloc allocator"
 license = "MIT"
 
 [dependencies]
-libc = "0.2"
 
 [build-dependencies]
 cmake = "0.1"

--- a/libmimalloc-sys/src/lib.rs
+++ b/libmimalloc-sys/src/lib.rs
@@ -1,14 +1,14 @@
 // Copyright 2019 Octavian Oncescu
 
-use libc::{c_void, size_t};
+use core::ffi::c_void;
 
 extern "C" {
-    pub fn mi_zalloc(size: size_t) -> *const c_void;
-    pub fn mi_malloc(size: size_t) -> *const c_void;
-    pub fn mi_realloc(p: *const c_void, size: size_t) -> *const c_void;
-    pub fn mi_zalloc_aligned(size: size_t, alignment: size_t) -> *const c_void;
-    pub fn mi_malloc_aligned(size: size_t, alignment: size_t) -> *const c_void;
-    pub fn mi_realloc_aligned(p: *const c_void, size: size_t, alignment: size_t) -> *const c_void;
+    pub fn mi_zalloc(size: usize) -> *const c_void;
+    pub fn mi_malloc(size: usize) -> *const c_void;
+    pub fn mi_realloc(p: *const c_void, size: usize) -> *const c_void;
+    pub fn mi_zalloc_aligned(size: usize, alignment: usize) -> *const c_void;
+    pub fn mi_malloc_aligned(size: usize, alignment: usize) -> *const c_void;
+    pub fn mi_realloc_aligned(p: *const c_void, size: usize, alignment: usize) -> *const c_void;
     pub fn mi_free(p: *const c_void) -> c_void;
 }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -27,7 +27,7 @@ extern crate libmimalloc_sys as ffi;
 
 use core::alloc::{GlobalAlloc, Layout};
 use ffi::*;
-use libc::c_void;
+use core::ffi::c_void;
 
 // Copied from https://github.com/rust-lang/rust/blob/master/src/libstd/sys_common/alloc.rs
 #[cfg(all(any(

